content/en/docs/languages/python/quickstart.md

```diff
@@ -189,8 +189,8 @@ def run():
         stub = helloworld_pb2_grpc.GreeterStub(channel)
         response = stub.SayHello(helloworld_pb2.HelloRequest(name='you'))
         print("Greeter client received: " + response.message)
-        response = stub.SayHelloAgain(helloworld_pb2.HelloRequest(name='you'))
-        print("Greeter client received: " + response.message)
+        rresponse2 = stub.SayHelloAgain(helloworld_pb2.HelloRequest(name='you'))
```

Comment:
`s/rresponse2/response2/`

Reply:
It is needed as the response object is being over written and it prints the same message again. It is confusing.

Reply:
I'm assuming this is in response to my more general comment on this PR rather than this comment pointing out the typo in the change.

Indeed. The response object is being overwritten. Overwritten with the new response from the `SayHelloAgain` method.

>  it prints the same message again
I think you should double-check that your server `SayHelloAgain` handler is returning the text you want it to.